labels.py

```diff
@@ -101,17 +101,25 @@ def get_stats_time_in_labels(
                 if issue.label_metrics[label] is None:
                     continue
                 if label not in time_in_labels:
-                    time_in_labels[label] = [issue.label_metrics[label]]
+                    time_in_labels[label] = [issue.label_metrics[label].total_seconds()]
```

Comment:
The rest of this PR is add `numpy.round()` and some cosmetic changes.

Just curious, why did you have to add the `.total_seconds` here?

Reply:
This was definitely a mistake. In fact if the `label` is not in `time_in_labels` then I don't think we should be doing anything! I'll fix the total_seconds part here and remove the whole line in a separate PR with some testing.

Reply:
Well, I was wrong. 😄 This is needed. This part of the code stores the timedelta information for a given label and instead of storing it as a timedelta object, i'm choosing to store it as a number representing total_seconds so that round and averaging is easier. It is later converted back into a timedelta object after the math operations are complete.